gui-apps/kanshi/metadata.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+    <longdescription>
+        kanshi allows you to define output profiles that
+        are automatically enabled and disabled on hotplug.
+        For instance, this can be used to turn a laptop's
+        internal screen off when docked.
+    </longdescription>
+    <maintainer type="person">
+        <email>gentoo@aisha.cc</email>
+        <name>Aisha Tammy</name>
+    </maintainer>
+    <maintainer type="project">
```

Comment:
Please remove from m-n. I will be the Gentoo maintainer and you can be the main contact.

Reply:
done.